src/main/java/net/minecraftforge/common/crafting/CraftingHelper.java

```diff
@@ -197,6 +197,49 @@ public static ItemStack getItemStack(JsonObject json, boolean readNBT)
 
         return new ItemStack(item, JSONUtils.getAsInt(json, "count", 1));
     }
+    
+    public static ItemStack getResultItemStack(JsonObject json, boolean readNBT)
+    {
+        String itemName = JSONUtils.getAsString(json, "result");
```

Comment:
Instead of duplicating the entire method to change one string, why not make a method `getItemStack(JsonObject json, boolean readNBT, String key)` then have the current header call that with `"item"`

Reply:
I also taught about doing that, but it would only really be useful if others would also call it. But making it so it _can_ be used is a good idea. I’m currently adding a method to the craftinghelper, so it can be used by others as well (like maybe cooking?). Or would it be better to not do that and just patch it for the stonecutter?

Reply:
Almost any time you are not duplicating code its useful, whether its a private helper or a public one. Currently if any change is made to one copy, you have to make it to both. So whether this method will be used in more places or not, don't duplicate a large chunk of code.